### PR TITLE
JBPM-6865 - Open/collapse styling for multi select filter options

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
jenkins please retest this

**Comment by bsremac** [comment]
It's not directly related, but it might make sense to fix this as part of this PR. Otherwise, I would have to open a dedicated JIRA bug for this.
We fixed the use case of clearing all selections in miltiselect filter on landing on the perspective, but ther's another use case which needs to be covered - when we do Clear All filters. In this case selections should also be cleared.

screenshot from 2018-03-23 11-24-41

File: jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/basic/BasicFiltersViewImpl.java [hunk 1]
```diff
@@ -339,8 +344,63 @@ public void addMultiSelectFilter(final String label,
...
+        final Div title = (Div) getDocument().createElement("div");
+        addCSSClass(title,
+                    "panel-title");
+        addCSSClass(title,
+                    "kie-dock__heading--section");
+        heading.appendChild(title);
+
+        final Anchor anchorTitle = (Anchor) getDocument().createElement("a");
+        anchorTitle.setAttribute("data-toggle",
+                                 "collapse");
```
[the remark this answers is not in the document]
Comment (cristianonicolai) [reply jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/filters/basic/BasicFiltersViewImpl.java]: Yeah, its really a Bootstrap feature...

**Comment by cristianonicolai** [comment]
@bsremac ok so fixed the clear all state issue. I also updated the behavior of the buttons to be disabled instead of hidden when no filters. Something UX has mentioned a few times.

**Pull request merged**